src/utils/__tests__/ga-ecomm-helpers.test.js

```diff
@@ -0,0 +1,42 @@
+const { filterEcommEvents, isEcommEvent } = require('../');
+
+describe('Util: Filter Ecommerce Events', () => {
+  const [name, id, revenue] = ['fooname', 'fooid', 523.55];
+  const baseEvent = {
+    hitType: 'addTransaction',
+    customTrackerId: 'myTracker',
+    id,
+    name,
+    revenue,
+  };
+
+  it('should return an object with hitType and trackerId stripped out', () => {
+    const result = filterEcommEvents(baseEvent);
+    const expected = {
+      id,
+      name,
+      revenue,
+    };
+    expect(result).toEqual(expected);
+  });
+
+  it('should return true if event hitType is related to ecommerce', () => {
+    const events = [
+      { hitType: 'addTransaction', id: 'myid' },
+      { hitType: 'addItem', id: 'myitemid', name: 'my item' },
+      { hitType: 'ecommClear' },
+      { hitType: 'ecommSend' },
+    ];
+    events.forEach(evt => expect(isEcommEvent(evt)).toBeTruthy());
+  });
+
+  it('should return false if event hitType is not related to ecommerce', () => {
+    const events = [
+      { hitType: 'pageview' },
+      { hitType: 'screenview' },
+      { hitType: 'event' },
+      { hitType: 'social' },
+    ];
+    events.forEach(evt => expect(isEcommEvent(evt)).toBeFalsy());
```

Comment:
:1st_place_medal: 

This is great, but one little thought I had is that when tests are written like this, its hard to tell what fails when it fails. I prefer to have one assertion per test. Like so:

```js
  [
    { hitType: 'addTransaction', id: 'myid' },
    { hitType: 'addItem', id: 'myitemid', name: 'my item' },
    { hitType: 'ecommClear' },
    { hitType: 'ecommSend' },
  ].forEach(event => {
    test(`${JSON.stringify(event)} => true`, () => {
      expect(isEcommEvent(event)).toBeTruthy();
    });
  });
```

In that way, when it fails, its obvious which scenario failed:

```
 ✕ {"hitType":"addTransaction","id":"myid"} => true
 ✓ {"hitType":"addItem","id":"myitemid","name":"my item"} => true (1ms)
 ✓ {"hitType":"ecommClear"} => true
 ✓ {"hitType":"ecommSend"} => true
```